webtop/__init__.py

```diff
@@ -28,7 +30,9 @@ def parse_args() -> argparse.Namespace:
         default="GET",
     )
 
-    parser.add_argument("-k", "--threads", metavar="N", type=int, help="Thread pool size", default=1)
+    parser.add_argument(
+        "-j", "-k", "--jobs", "--threads", dest="jobs", metavar="N", type=int, help="Number of jobs", default=1
```

Comment:
Why change the interface here? And why multiple args for the same function?

Reply:
Threads is now misleading. I kept `-k` and `--threads` for backwards compatibility with scripts or bash history, but those should eventually be deprecated.

Reply:
asyncio is still using threads under the hood, so it's not misleading. and `-k` is a common flag in CLI programs to control number of threads.

`job` is arguably a much more vague word than `thread`

Reply:
```bash
# Using master branch, python3 -m http.server 8000 in other terminal
python webtop/__init__.py http://127.0.0.1:8000/ -k 5 & pid=$!
ps M $pid
# USER      PID   TT   %CPU STAT PRI     STIME     UTIME COMMAND
# efoster 11563 s000    0.0 S    20T   0:00.06   0:00.26 /usr/local/Cellar/python/3.7.4/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python webtop/__init__.py http://127.0.0.1:8000/ -k 5
#         11563         2.7 S    20T   0:00.03   0:00.06
#         11563        13.6 R    20T   0:00.11   0:00.66
#         11563        14.2 S    20T   0:00.11   0:00.65
#         11563        12.5 S    20T   0:00.11   0:00.64
#         11563        12.6 S    20T   0:00.11   0:00.69
#         11563        12.1 S    20T   0:00.11   0:00.65
#         11563         0.0 S    20T   0:00.00   0:00.00

# Using branch asyncio
python webtop/__init__.py http://127.0.0.1:8000/ -k 5 & pid=$!
ps M $pid
# USER      PID   TT   %CPU STAT PRI     STIME     UTIME COMMAND
# efoster 11890 s000   45.0 R    20T   0:00.76   0:02.41 /usr/local/Cellar/python/3.7.4/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python webtop/__init__.py http://127.0.0.1:8000/ -k 5
```

asyncio doesn't use threads. `aiohttp` is completely single threaded, and all context switching is done cooperatively and explicitly using `await`, `async for`, `async while`, and `async with`. You *can* use threads with `asyncio` by doing `loop.run_in_executor`, but pure asyncio code is 100% single threaded.

Reply:
I won't argue if jobs is more vague, but `thread` is completely incorrect in this case.

Reply:
Possibly `-k` and `--workers` ? I based `-j` off of `make`, but I'm not married to it.

Reply:
`workers` sounds like a good compromise

Reply:
Should I keep `--threads` for backwards compatibility? Or will `-k` be enough? I don't know if anyone has written scripts to use this yet so maybe it doesn't matter.

Reply:
I'm gonna guess no and I'll just fix it later if need be.